h/browser/chrome/lib/sidebar-injector.js

```diff
@@ -288,6 +308,22 @@ function SidebarInjector(chromeTabs, dependencies) {
   function injectScript(tabId, path) {
     return executeScriptFn(tabId, {file: path});
   }
+
+  /**
+   * Inject configuration information for the Hypothesis application
+   * into the page as JSON data via a <meta> tag.
+   *
+   * A <meta> tag is used because that makes it available to JS content
+   * running in isolated worlds.
+   */
+  function injectConfig(tabId, config) {
```

Comment:
I have reworked this now to use the same mechanism but look for a `js-hypothesis-config` class on the `<script>` tag instead of `js-hypothesis-settings`.